install/omv-install.sh

```diff
@@ -23,25 +23,35 @@ msg_ok "Installed Dependencies"
 msg_info "Installing OpenMediaVault (Patience)"
 wget -qO- https://packages.openmediavault.org/public/archive.key | gpg --dearmor >"/etc/apt/trusted.gpg.d/openmediavault-archive-keyring.gpg"
 
-cat <<EOF >/etc/apt/sources.list.d/openmediavault.list
-deb https://packages.openmediavault.org/public shaitan main
-# deb https://downloads.sourceforge.net/project/openmediavault/packages shaitan main
+cat <<EOF >> /etc/apt/sources.list.d/openmediavault.list
+deb [signed-by=/usr/share/keyrings/openmediavault-archive-keyring.gpg] http://packages.openmediavault.org/public sandworm main
+# deb [signed-by=/usr/share/keyrings/openmediavault-archive-keyring.gpg] http://downloads.sourceforge.net/project/openmediavault/packages sandworm main
 ## Uncomment the following line to add software from the proposed repository.
-# deb https://packages.openmediavault.org/public shaitan-proposed main
-# deb https://downloads.sourceforge.net/project/openmediavault/packages shaitan-proposed main
+# deb [signed-by=/usr/share/keyrings/openmediavault-archive-keyring.gpg] http://packages.openmediavault.org/public sandworm-proposed main
+# deb [signed-by=/usr/share/keyrings/openmediavault-archive-keyring.gpg] http://downloads.sourceforge.net/project/openmediavault/packages sandworm-proposed main
 ## This software is not part of OpenMediaVault, but is offered by third-party
 ## developers as a service to OpenMediaVault users.
-# deb https://packages.openmediavault.org/public shaitan partner
-# deb https://downloads.sourceforge.net/project/openmediavault/packages shaitan partner
+# deb [signed-by=/usr/share/keyrings/openmediavault-archive-keyring.gpg] http://packages.openmediavault.org/public sandworm partner
+# deb [signed-by=/usr/share/keyrings/openmediavault-archive-keyring.gpg] http://downloads.sourceforge.net/project/openmediavault/packages sandworm partner
 EOF
 
 export LANG=C.UTF-8
 export DEBIAN_FRONTEND=noninteractive
 export APT_LISTCHANGES_FRONTEND=none
-$STD apt-get update
-apt-get -y --auto-remove --show-upgraded --allow-downgrades --allow-change-held-packages --no-install-recommends --option DPkg::Options::="--force-confdef" --option DPkg::Options::="--force-confold" install openmediavault-keyring openmediavault &>/dev/null
+apt-get install --yes gnupg
+wget --quiet --output-document=- https://packages.openmediavault.org/public/archive.key | gpg --dearmor --yes --output "/usr/share/keyrings/openmediavault-archive-keyring.gpg"
```

Comment:
long flags are not used in these scripts (`--quiet`)

Hold on, you've got the sequence messed up

Reply:
> long flags are not used in these scripts (`--quiet`)
> 
> Hold on, you've got the sequence messed up

Actually you catched me when I was testing + copy pasting from other repos ( SourceForge etc.. ) to check them them :D
Will clean up the script and add a little tomorrow, now I'm testing locally to not confuse you with Gitub's autamtic notifications...